packages/aws-cdk-lib/aws-ses/test/configuration-set-event-destination.test.ts

```diff
@@ -105,3 +106,42 @@ test('cloudwatch dimensions destination', () => {
   });
 });
 
+test('event bridge destination', () => {
```

Comment:
Can you please add a test to validate the `Only the default bus can be used as an event destination` exception being thrown?